### DISCUSSION TITLE
Could the results of MD with enhanced sampling be used as training data for DEEPMD

### DISCUSSION THREAD

**Discussion by hahha2022** [deepmodeling/deepmd-kit#1940] [Q&A] [answered]
For example, I run  a MD sitlamution with slowing-groth method(continuously reduce CV value during MD simulation) in Vasp,  could the results (OUTCAR) of this MD sitlamution be used as training data for DEEPMD? As far as I understand, in the process of enhanced sampling, the fixed atoms may get incorrect force or energy due to the additional fixed force, is that right?

**Comment by njzjz** [comment, marked as answer]
I think you can validate it by performing single-point calculations. If different, use the results from single-point calculations.

**Reply by hahha2022** [reply]
Thank you very much. After my testing, I found that the fixed atoms indeed get incorrect energy due to the additional fixed force on the CV.
